### PR TITLE
[Feature]: Awards page(s)

### PR DESCRIPTION
**Pull request by mkanoria**
- UI for the Awards & Awards Detail Page

Most of the UI is done, filtering should be added in when API hookups are done. Data is static for now but will be updated to dynamic once API's are conected

*Note* - A lot of this is content from the performances pages so there might be some overlap there

image

image

image

### CODE REVIEW

File: pages/awards/[id].js [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+      </div>
+    </Layout>
+  );
+}
+
+function Tab(props) {
+  const handleClick = () => {
+    props.setSelectedTab(props.tabIndex);
+    props.handleClick();
+  };
```
Comment (OustanDing) [inline pages/awards/[id].js:101]: Can we destructure props here so we can use PropTypes? We can also do it during the API hookup.

File: pages/awards/index.js [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+          default:
+            throw new Error('Invalid setting type');
+        }
+      }
+
+      setAcademicLevelOptions(academicLevelSettings);
+    };
+
+    getSettings();
+  }, []);
```
Comment (OustanDing) [inline pages/awards/index.js:91]: We can just remove this commented code. I refactored it in https://github.com/uwblueprint/dancefest-web/pull/212.

File: styles/components/Layout.module.scss [hunk 3]
```diff
@@ -86,7 +86,7 @@ Layout: Content
   min-height: calc(100vh - 150px);
 
   .layout__content_sizer {
-    max-width: 1200px;
+    max-width: 1350px;
```
Comment (OustanDing) [inline styles/components/Layout.module.scss:89]: Wasn't this reverted to `1200px`?

**Review by OustanDing — COMMENTED** [review]
Looks pretty good overall. Let's split up the components in the awards pages into their own files. Performances got pretty messy when we had everything in one file.

**Review by thepenguinco — APPROVED** [review]
LGTM.  Let's do integration testing on staging.

File: pages/awards/index.js [hunk 4]
```diff
@@ -0,0 +1,311 @@
...
+      Header: 'No. of nominations',
+      accessor: 'nominations',
+      filter: 'matchEnum',
+      headerStyle: { textAlign: 'right' },
+    },
+  ];
+
+  const goToPerformanceDetails = row => {
+    // Go to /performances/[id] page
+    console.log(row);
```
Comment (OustanDing) [inline pages/awards/index.js:235]: Remove `console.log`

**Review by OustanDing — APPROVED** [review]
LGTM!

**Pull request merged**